ydb/docs/en/core/reference/docker/install.md

```diff
@@ -0,0 +1,11 @@
+# Installing Docker to work with {{ ydb-short-name }}
```

Comment:
Rename this file to `prerequisites.md` because it currently sounds like this is about installing YDB and not Docker.